kie-spring-boot/kie-spring-boot-samples/jbpm-spring-boot-sample-basic/src/main/java/org/jbpm/springboot/samples/WebSecurityConfig.java

```diff
@@ -33,8 +36,15 @@ protected void configure(HttpSecurity http) throws Exception {
     }
 
     @Autowired
-    public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {
+    public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {        
+        PasswordEncoder encoder = PasswordEncoderFactories.createDelegatingPasswordEncoder();
+              
         auth.inMemoryAuthentication()
-                .withUser("john").password("john1").roles("jbpm,HR,IT,Accounting,PM");
+                .withUser("john").password(encoder.encode("john1")).roles("jbpm,HR,IT,Accounting,PM");
+    }
+    
+    @Bean
+    public PasswordEncoder passwordEncoder() {
+        return PasswordEncoderFactories.createDelegatingPasswordEncoder();
```

Comment:
As I understand it now, this bean declaration is not needed since we don't inject it anywhere. I tried it without it and I was able to log in correctly.

Reply:
fixed